src/components/ChooseOfferForm/ChooseOfferForm.tsx

```diff
@@ -24,15 +24,37 @@ type Props = {
   onBackButtonClickHandler?: () => void;
   monthlyOffer?: Offer;
   yearlyOffer?: Offer;
+  tvodOffers: Offer[];
   submitting: boolean;
 };
 
-const ChooseOfferForm: React.FC<Props> = ({ values, errors, onChange, onSubmit, submitting, yearlyOffer, monthlyOffer, onBackButtonClickHandler }: Props) => {
+type OfferBoxProps = {
+  offer: Offer;
+  radioKey: 'periodicity' | 'tvodOfferId';
+  radioValue: string;
+  title: string;
+  ariaLabel: string;
+  secondBenefit?: string;
+  periodString?: string;
+};
+type OfferBox = ({ offer, radioKey, radioValue, title, ariaLabel, secondBenefit, periodString }: OfferBoxProps) => JSX.Element;
+
+const ChooseOfferForm: React.FC<Props> = ({
+  values,
+  errors,
+  onChange,
+  onSubmit,
+  submitting,
+  yearlyOffer,
+  monthlyOffer,
+  tvodOffers,
+  onBackButtonClickHandler,
+}: Props) => {
   const { siteName } = useContext(ConfigContext);
   const { t } = useTranslation('account');
 
   const getFreeTrialText = (offer: Offer) => {
-    if (offer.freeDays > 0) {
+    if (offer.freeDays && offer.freeDays > 0) {
```

Comment:
`freeDays` can actually be `null` (I had to update the types), in which case the `> 0` isn't valid. 
I could change it to `Number(offer.freeDays) > 0`, if you find that more elegant, since `Number(null)` casts to 0.

Reply:
Why not just `offer.freeDays`?

Reply:
"Object is possibly 'null'"

![Screenshot 2022-05-24 at 11 05 17](https://user-images.githubusercontent.com/61868085/169994510-adaf0982-de52-4deb-97aa-8e8414f71e69.png)

Reply:
Excuse me, of course!